idp-stepup-api/pom.xml

```diff
@@ -1,48 +1,62 @@
-<!-- The MIT License Copyright (c) 2015, 2019 CSC - IT Center for Science, http://www.csc.fi Permission is hereby granted, free 
-    of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in 
-    the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, 
-    sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject 
-    to the following conditions: The above copyright notice and this permission notice shall be included in all copies or substantial 
-    portions of the Software. THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT 
-    NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL 
-    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT 
-    OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>idp-stepup-api</artifactId>
-    <parent>
-        <groupId>fi.csc.idp.stepup</groupId>
-        <artifactId>idp-stepup-parent</artifactId>
-        <version>0.10.0</version>
-        <relativePath>..</relativePath>
-    </parent>
-    <packaging>jar</packaging>
-    <name>idp-stepup-api</name>
-    <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>net.shibboleth.idp</groupId>
-            <artifactId>idp-authn-api</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>com.nimbusds</groupId>
-            <artifactId>oauth2-oidc-sdk</artifactId>
-        </dependency>
-    </dependencies>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>${maven.compiler.source}</source>
-                    <target>${maven.compiler.target}</target>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+<?xml version="1.0"?>
+<!-- The MIT License Copyright (c) 2015, 2019 CSC - IT Center for Science, 
```

Comment:
```suggestion
<!-- The MIT License Copyright (c) 2015 - 2020 CSC - IT Center for Science, 
```

Reply:
There are certainly also other license year misses. I can fix them all based on this comment.